test/jest.src.config.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   rootDir: '../src',
+  collectCoverage: true,
   testEnvironment: 'node',
-  collectCoverageFrom: ['src/**/*.js'],
```

Comment:
Why did you make these changes?

Reply:
I'm guessing how `jest-cli` resolves paths for coverage changed somehow because with this line it wasn't tracking coverage at all.

Reply:
Huh.... Ok, that's fine. I plan to eventually upgrade this package to use [kcd-scripts](https://github.com/kentcdodds/kcd-scripts) anyway.

Thanks!

Reply:
I know this has already been merged, but as an addition.

Changing the jest config to the following seems to have the 
same effect as removing the `collectCoverageFrom` line.

```js
module.exports = {
  // ...
  collectCoverageFrom: ['**/*.js'],
  coveragePathIgnorePatterns: [
    '/coverage/',
    // ...
  ],
  // ...
}
```

Reply:
Thanks. I'd rather just upgrade this to use kcd-scripts instead.